backend/de.metas.externalreference/src/main/java/de/metas/externalreference/rest/v1/ExternalReferenceRestController.java

```diff
@@ -43,8 +43,7 @@
 
 @RequestMapping(value = {
 		MetasfreshRestAPIConstants.ENDPOINT_API_DEPRECATED + "/externalRef",
-		MetasfreshRestAPIConstants.ENDPOINT_API_V1 + "/externalRef",
-		MetasfreshRestAPIConstants.ENDPOINT_API_V2 + "/externalRefs" })
+		MetasfreshRestAPIConstants.ENDPOINT_API_V1 + "/externalRef" })
 @RestController
 @Profile(Profiles.PROFILE_App)
```

Comment:
```suggestion
@Profile(Profiles.PROFILE_App)
@Deprecated
```

(let's flag everything `v1` as deprecated, at least when we touch it)